### ISSUE TITLE
Missing build instruction for QT-Creator

### ISSUE THREAD

**Issue by orophix** [herace/AnimeEffects#21]
Ubuntu 20.04

Unabable to build, if I try to open AnimeEffects.pro as a project I get:

`No build found in /home/orophix/Tools/AnimeEffects/src matching project /home/orophix/Tools/AnimeEffects/src/AnimeEffects.pro.`

[Link to Screenshot](https://nyah.moe/img/RDk8q)

**Comment by Jose-Moreno** [comment]
@orophix Hey. Sorry about the late reply, I saw this early and shared your question with some of the contributors, these are some of their replies. I too concur that you would probably to thoroughly reinstall the qt framework, however I'm using Windows so I can't help there much more:

> azagaya
> Thats probably a problem with qt installation
> Kits are different qt versions or config that one should configure in their computer. If you use the online installer kits are set automatically iirc
> For ubuntu there is a ppa from _stephen binner_
> That is quite up to date even for xenial
> I use it to package the appimage

> WolfyHawk
> he is missing `sudo apt install qt5-default`
> but the default version is to low to build AE... (5.12.8)

_Note: Apparently you require at least 5.14 to build AnimeEffects properly_

> Larpon
> I usually don't recommend using the system provided Qt packages since they're always outdated in some manner. It always leads to confusion and sometimes Qt Creator doesn't detect them properly etc.

Let's tag @herace to see if they have any other suggestion regarding this particular issue as well 🙂 

Lastly If you still can't solve this, please come to the [discord server we've setup for AnimeEffects](https://discord.gg/sKp8Srm) and chat with either of them in the `core-dev channel` to see if it's possible to solve this problem of yours. Cheers.

**Comment by herace** [comment]
Yeah, i expected that just installing qtcreator wasn't  enough but went ahead and added @freddii because having some instructions is better then nothing. 

@orophix I went ahead and update the installation instructions, give it a try and see if that solve your issue, also like Jose suggested go on our discord.

@Jose-Moreno Yeah, im sure you know this but Debian based distros are not rolling so running into these problems is very common and honestly annoying, hence why i prefer using a rolling release distro like arch 

Btw i added arch installation instructions as well and tested it on a clean Manjaro install just now so i can say its should be fine.
